angel.py

```diff
@@ -171,27 +174,20 @@ def write_to_csv(index, name01, *player_lists):
             cur_time = time.strftime("%Y-%m-%d %H-%M-%S")
             with open(f"{index} - {name01} - {cur_time}.csv", 'w', newline='') as f: # In Python 3, if do not put newline='' AND choose 'w' instead of 'wb', you will have an empty 2nd row in output .csv file.
                 writer = csv.writer(f, delimiter=',')
-                header = ['Telegram Username','Name','GenderPref','Gender','Interests','2truths1lie','Intro','House','CG','Year','Faculty'] # add header to output csv file
+
+                config_file = open("config.yml", 'r')
+                index_dict = yaml.full_load(config_file)["player_attribute_index"]
+                index_lst = {k: v for k, v in sorted(index_dict.items(), key=lambda item: item[1])}
+
+                header = [] # add header to output csv file
+                for key, value in index_lst.items():
+                    string2 = key.replace('_', " ")
+                    string3 = ''.join(string2)
+                    string4 = string3.title()
+                    header.append(string4)
+
                 writer.writerow(i for i in header)
                 for player in player_list:
-                    if '\n' in player.twotruthsonelie:
-                        string1 = player.twotruthsonelie
-                        string2 = string1.replace('"', "'")  # JUST IN CASE PEOPLE TYPE " which can screw up a csv file
-                        string3 = ''.join(('"', string2,'"'))  # Double quotations are what CSV uses to keep track of newlines within the same cell
```

Comment:
Even though we removed these columns, there are still text columns in tAngels. Did you validate that these checks are no longer necessary? It's true that this can be abstracted into a function run on all text columns.